spyderlib/widgets/sourcecode/syntaxhighlighters.py

```diff
@@ -414,6 +414,20 @@ def highlightBlock(self, text):
                     
             match = self.PROG.search(text, match.end())
  
+        # Make blank space less apparent by setting the foreground alpha.

```

Comment:
All this should have an if statement tied to the option of showing spaces, no?

Also to make it work with other syntax highlighters it should be refactored as a separate method inside BaseSH